app/src/main/java/com/codingblocks/cbonlineapp/activities/CourseActivity.kt

```diff
@@ -255,6 +240,32 @@ class CourseActivity : AppCompatActivity(), AnkoLogger {
         })
     }
 
+    private fun fetchTags(course: Course) {
+
+        course.runs?.forEach {
+            if(it.tags?.size==0){
+                tagstv.visibility = View.GONE
+                coursePagevtags.visibility = View.GONE
+                tagsChipgroup.visibility = View.GONE
+            }else{
+                tagstv.visibility = View.VISIBLE
+                coursePagevtags.visibility = View.VISIBLE
+                tagsChipgroup.visibility = View.VISIBLE
+
+                it.tags?.forEach {
+                    val chip  = Chip(this)
+                    chip.text = it.name
+                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
Don't use this check make it work for devices starting from API > 21

Reply:
Okay